pyproject.toml

```diff
@@ -16,25 +16,25 @@ classifiers=[
         'Programming Language :: Python :: 3.8',
         'Programming Language :: Python :: 3.9',
         'Programming Language :: Python :: 3.10',
+        'Programming Language :: Python :: 3.11',
+        'Programming Language :: Python :: 3.12',
         'Programming Language :: Python :: Implementation :: CPython'
     ]
 
 [tool.poetry.dependencies]
-python = ">=3.8,<3.11"
-certifi = ">=2017.4.17"
-six = ">=1.10"
-python-dateutil = ">=2.5.3"
-urllib3 = ">=1.26.7"
-tzlocal = ">=4.0.0,<5.0.0"
+python = ">=3.8,<=3.12"
+certifi = "^2023.11.17"
+six = "^1.16"
+python-dateutil = "^2.7.2"
+urllib3 = "<2.1.0"
```

Comment:
We can't update to versions 2.1.0+ until [CIAC-9648](https://jira-dc.paloaltonetworks.com/browse/CIAC-9648) is resolved.